design-documents/frontend/admin-guidelines.md

```diff
@@ -0,0 +1,215 @@
+# Magento Admin Guidelines
+
+## Problem
+
+There are already a handful of React applications built for Magento Admin use, but they have different dependencies and versions between them and we don't yet have a micro front-end architecture to manage them.
```

Comment:
@jedmao does it mean that upgrade of Magento Admin frontend (currently, UI component-based) is somewhere on the horizon?

Reply:
There are no plans for that as of yet.